sqlup-mode.el

```diff
@@ -67,19 +67,12 @@
 this mode's logic will be evaluated.")
 
 (defconst sqlup-eval-keywords
-  '((postgres "EXECUTE"))
-"List of keywords introducing eval strings, organised by dialect.")
+  '((postgres "EXECUTE" "format("))
```

Comment:
This is because you can use either a plain EXECUTE with parameters or EXECUTE format(), with parameters as well as generated names for tables, views, etc. They are two slightly different things.